comp/metadata/softwareinventory/impl/inventorysoftware.go

```diff
@@ -134,48 +143,112 @@ func newWithClient(reqs Requires, client sysProbeClient) (Provides, error) {
 		log:            reqs.Log,
 		sysProbeClient: client,
 		hostname:       hname,
+		eventPlatform:  reqs.EventPlatform,
 	}
 
-	// Note that there is a second way to disable this feature, through InventoryPayload.Enabled.
-	// 'enable_metadata_collection' and 'inventories_enabled' both need to be set to true.
 	if !reqs.Config.GetBool("software_inventory.enabled") {
 		return Provides{
 			Comp: is,
 		}, nil
 	}
 
+	localSource := rand.NewSource(time.Now().UnixNano())
+	localRand := rand.New(localSource)
+
+	is.jitter = time.Duration(localRand.Intn(max(reqs.Config.GetInt("software_inventory.jitter"), 60))) * time.Second
+	is.interval = time.Duration(max(reqs.Config.GetInt("software_inventory.interval"), 10)) * time.Minute
```

Comment:
I think we already have settings for this: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/setup/config.go#L1017-L1020. Do we need different ones just for softwareinventory ?